python/ray/worker.py

```diff
@@ -311,10 +311,6 @@ def get_objects(self, object_ids, timeout=None):
                     "which is not an ray.ObjectID.".format(object_id))
 
         if self.mode == LOCAL_MODE:
-            # TODO(ujvl): Remove check when local mode moved to core worker.
-            if timeout is not None:
-                raise ValueError(
-                    "`get` must be called with timeout=None in local mode.")
```

Comment:
can you give me context for this change?

Reply:
i added it in the ray.get PR but realized it actually isn't necessary—for local mode it's just going to return immediately.